kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -940,6 +1010,10 @@ public Optional<TimePartitioning.Type> getTimePartitioningType() {
     return parseTimePartitioningType(getString(TIME_PARTITIONING_TYPE_CONFIG));
   }
 
+  public Optional<Map<String, String>> getTopic2TableMap(boolean sanitize) {
```

Comment:
may be thinking loud, sanitize is passed as param instead of reading directly from config itself.  is that considered for better readability so that if somebody read sinkTask then dev can easily interpret getTopic2TableMap also depends on sanitize value.

Reply:
Oh I just did that because sanitize was read and set using config a few lines above.     
`sanitize = config.getBoolean(BigQuerySinkConfig.SANITIZE_TOPICS_CONFIG);` I thought it might be ok. Let me know if you think otherwise.